frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -1,46 +1,170 @@
 
 interface TableProps {
-  data: HealthDataForArea[];
+  data: HealthDataForArea;
   headings: string[];
 }
 
+interface LineChartTableData {
+  period: number;
+  count: number;
+  value: number;
+  lower: number;
+  upper: number;
+}
+
+const LIGHT_GREY = '#cbcbcb';
+
+const StyledTableCellHeader = styled(Table.CellHeader)(
+  typography.font({ size: 16 }),
+  {
+    fontWeight: 'bold',
+  },
+  {
+    paddingTop: '0.3125em',
+  }
+);
+
+const StyledAlignCenterHeader = styled(StyledTableCellHeader)({
+  textAlign: 'center',
+});
+
+const StyledAlignRightHeader = styled(StyledTableCellHeader)({
+  textAlign: 'right',
+  paddingRight: '0',
+});
+
+const StyledAlignLeftHeader = styled(StyledTableCellHeader)({
+  textAlign: 'left',
+});
+
+const StyledConfidenceLimitsHeader = styled(StyledTableCellHeader)({
+  padding: '0.625em',
+  paddingLeft: '0',
+});
+
+const StyledGreyHeader = styled(StyledAlignRightHeader)({
+  backgroundColor: LIGHT_GREY,
+});
+
+const StyledTable = styled(Table)({
+  marginBottom: '0',
+});
+
+const StyledTableCell = styled(Table.Cell)(typography.font({ size: 16 }), {
+  paddingRight: '0',
+});
+
+const StyledAlignLeftTableCell = styled(StyledTableCell)({
+  textAlign: 'left',
+});
+
+const StyledAlignRightTableCell = styled(StyledTableCell)({
+  textAlign: 'right',
+});
+
+const StyledBenchmarkValueTableCell = styled(StyledTableCell)({
+  backgroundColor: LIGHT_GREY,
+  borderTop: 'solid white 0.09375em',
+});
+
+const mapToTableData = (areaData: HealthDataForArea): LineChartTableData[] =>
+  areaData.healthData.map((healthPoint) => ({
+    period: healthPoint.year,
+    count: healthPoint.count,
+    value: healthPoint.value,
+    lower: healthPoint.lowerCi,
+    upper: healthPoint.upperCi,
+  }));
+
+function getCellHeader(heading: string): ReactNode {
+  return heading === LineChartTableHeadingEnum.BenchmarkValue ? (
+    <StyledGreyHeader key={`header-${heading}`}>{heading}</StyledGreyHeader>
+  ) : (
+    <StyledAlignRightHeader key={`header-${heading}`}>
+      {heading}
+    </StyledAlignRightHeader>
+  );
+}
+
 export function LineChartTable({ data, headings }: Readonly<TableProps>) {
-  data = sortHealthDataByDate(data);
+  const tableData = mapToTableData(data);
+  const dataSortedByPeriodDesc = tableData.toSorted(
+    (a, b) => b.period - a.period
+  );
   return (
     <div data-testid="lineChartTable-component">
-      <Table
+      <StyledTable
+        head={
+          <Table.Row>
+            <StyledAlignCenterHeader setWidth="one-quarter"></StyledAlignCenterHeader>
+            <StyledAlignCenterHeader setWidth="one-quarter"></StyledAlignCenterHeader>
+            <StyledAlignCenterHeader>{data.areaCode}</StyledAlignCenterHeader>
+            {[...Array(7).keys()].map((i) => (
+              <Table.CellHeader key={i}></Table.CellHeader>
+            ))}
+            <StyledGreyHeader>England</StyledGreyHeader>
+          </Table.Row>
+        }
+      ></StyledTable>
+      <StyledTable
```

Comment:
Ah I see - you have multiple tables. Whereas it should just be one.  This will start causing alignment issues 

What you can do is use `colspan` to make certain cells cover more than one cell. So for example the cell with the areaName and England. The areaName cell will have a colspan of 7. 

Sorry I didn't pick up on this before